File: BaekJoon/Java/10817_세수_Sorting.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

// 세수 10817 Sorting
// O( N logN )


public class Main {
    static int[] arr = new int[3];

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        for (int i = 0; i < 3; ++i) {
            arr[i] = Integer.parseInt(st.nextToken());
        }

        Arrays.sort(arr);

        System.out.println(arr[1]);
    }
}

File: BaekJoon/10817_세수_Sorting.cpp
#include <iostream>
#include <array>
#include <algorithm>

using namespace std;

// 세수 10817 Sorting
// O( N logN )


int main(void)
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    cout.tie(nullptr);

    array<int, 3> arr;

    for (int& x : arr) cin >> x;

    sort(arr.begin(), arr.end());

    cout << arr[1];


    return 0;
}
